src/Cylinder.cc

```diff
@@ -102,15 +102,15 @@ Errors Cylinder::Load(ElementPtr _sdf)
     {
       errors.push_back({ErrorCode::ELEMENT_INVALID,
           "Invalid <radius> data for a <cylinder> geometry. "
-          "Using a radius of 1."});
+          "Using a radius of 0.5."});
```

Comment:
Oh, I made the suggestion above before seeing this. Another reason :slightly_smiling_face:

Reply:
6169fec